.github/workflows/lint.yml

```diff
@@ -86,6 +86,8 @@ jobs:
                      --skip-dir test_driver \
                      --known-failure app/app-platform/ContentApp.cpp \
                      --known-failure app/app-platform/ContentApp.h \
+                     --known-failure app/app-platform/ContentAppClientCommandSender.cpp \
+                     --known-failure app/app-platform/ContentAppClientCommandSender.h \
```

Comment:
Please don't add new known-failure things.  Add new files to build rules properly.  The only reason this list is here is so @andy31415 did not have to have a huge PR touching code that neither he nor the reviewers of his PR really understood.

This is the part that gets "changed requested".

Reply:
This could be a path-rule if "not using GN" applies. 

However I would feel more comfortable if these would then be placed somewhere else than `src/app` since otherwise it causes confusion as `src/app` is generally managed by GN.

Reply:
removed these new files